tests/kml_test.py

```
@@ -186,6 +187,79 @@ def test_kml_parse(self) -> None:
+                  ),
+              ],
+          )
+
+        file_path = KMLFILEDIR / "output.kml"
+        doc.write(file_path=file_path, prettyprint=True, xml_declaration=False)
+
+        assert file_path.is_file(), "KML file was not created."
+
+        parsed_doc = kml.KML.parse(file_path)
+
+        assert parsed_doc.to_string() == doc.to_string(), "Written and original documents don't match"
```

Ruff

Line too long (102 > 88)

(E501)